sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -337,6 +342,8 @@ SELECT
     ) AS reported_first_session_ping,
     COALESCE(_previous.metadata.reported_metrics_ping, FALSE)
     OR COALESCE(_current.metadata.reported_metrics_ping, FALSE) AS reported_metrics_ping,
+    COALESCE(_previous.metadata.reported_baseline_ping, FALSE)
```

Comment:
Could we add [unitest](https://github.com/mozilla/bigquery-etl/tree/main/tests/sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1) for this added column?

Reply:
I've adjusted the tests to check this new column.  Does that work?

Reply:
LGTM